src/constants.js

```diff
@@ -0,0 +1,10 @@
+export const SERVICE_CONSTSANSTS = Object.freeze({
+  DELIMITER: ',',
+  MINIMAL_CAR_NAME_LENGTH: 1,
+  MAXIMAL_CAR_NAME_LENGTH: 5,
+  MINIMAL_CAR_COUNT: 2,
+  MOVING_REFERENCE_NUMBER: 4,
+  MOVING_INDICATOR: '-',
+});
+
+export const ERROR_PREFIX = '[ERROR]';
```

Comment:
Error 상황의 메세지를 전부 옮겨 오는 것은 어떤가요??

```suggestion
export const ERROR_PREFIX = {
NOT_NUMBER : '[ERROR] 숫자만 입력해주세요.'

--- 생략---
}
```

Reply:
메세지 모두 상수로 빼는 부분에 있어서 해당 메세지들이 다른 부분에 활용되지 않는 다는 점 때문에 굳이 분리하지 않았습니다..!
상수 분리 과정에서 제일 고민되는 부분이었는데 바로 피드백 주셨군요..! 감사합니다!